<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>process</class>
 <widget class="QWidget" name="process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duplicate Finder</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QProgressBar" name="progress_bar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="delete_selected_button">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="result_tree">
     <column>
      <property name="text">
       <string>Result</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="errors_tree">
     <column>
      <property name="text">
       <string>Errors</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="return_button">
     <property name="text">
      <string>Return to Main Menu</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
